### DISCUSSION TITLE
Using Blocks with Vite

### DISCUSSION THREAD

**Discussion by wirago** [shadcn-ui/ui#3316] [Q&A]
The Blocks dashboards would be a great way to start, but obviously they base on Next.js given the imports 
`import Image from "next/image"`

Since I am using Vite, the imports are not working and part of my current installation.
What is the recommended way to use the Blocks code? Can they somehow be imported like the other shadcn components?

Or do I just grab them from a Next.js setup?

**Comment by MattKonop** [comment]
Blocks uses a few of the Next.JS components, I had switched them out for react components or in the case of Image for the vanilla HTML <img/> element. Not sure if that caused an issue, but I then got an error when building with rollup. 
"Error when using sourcemap for reporting an error: Can't resolve original location of error."
